scripts/circle-ci/run-circle-tests.sh

```diff
@@ -73,8 +75,11 @@ JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xlog:class+unload=off"
 JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xlog:gc:build-%t-%p.gc.log"
 
 # External builds have a 16gb limit.
-if [ "$test_suite_index" -eq "13" ]; then
+if [ "$test_suite_index" -eq "14" ]; then
     export _JAVA_OPTIONS="-Xms2g -Xmx4g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
+elif [ "$test_suite_index" -eq "3" ]; then
+    export _JAVA_OPTIONS="-Xmx8g ${JAVA_GC_LOGGING_OPTIONS}"
```

Comment:
Oh, min == max would be good here. I also suspect it may make the runtimes of CI more deterministic

For, active processor count, I am not familiar with more internals. From the documentation, it seems reasonable to set this https://docs.oracle.com/en/java/javase/11/tools/java.html#GUID-3B1CE181-CD30-4178-9602-230B800D4FAE:~:text=Overrides%20the%20number%20of%20CPUs%20that%20the%20VM%20will%20use%20to%20calculate%20the%20size%20of%20thread%20pools%20it%20will%20use%20for%20various%20operations%20such%20as%20Garbage%20Collection%20and%20ForkJoinPool. (though it does look more useful for if we had multiple JVMs runs in addition to this one

But, will push through with it eventually. Testing with multiple runs costs more than just merge and observe at this point

Thanks for the pointers!